Directory.Packages.props

```diff
@@ -0,0 +1,25 @@
+<Project>
+	<ItemGroup>
+		<PackageVersion Include="AMQPNetLite" Version="2.4.11" />
+ 		<PackageVersion Include="AMQPNetLite.Serialization" Version="2.4.11" />
+ 		<PackageVersion Include="Apache.Avro" Version="1.11.3" />
+ 		<PackageVersion Include="Confluent.Kafka" Version="1.9.3" />
+ 		<PackageVersion Include="Google.Protobuf" Version="3.27.3" />
+ 		<PackageVersion Include="McMaster.Extensions.CommandLineUtils" Version="4.1.1" />
+ 		<PackageVersion Include="Microsoft.AspNetCore.Http" Version="2.1.34" />
+ 		<PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="6.0.32" />
+ 		<PackageVersion Include="Microsoft.CSharp" Version="4.7.0" />
+ 		<PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+		<PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
+ 		<PackageVersion Include="MQTTnet" Version="4.3.6.1152" />
+ 		<PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+ 		<PackageVersion Include="Nullable" Version="1.3.1" />
+ 		<PackageVersion Include="System.Memory" Version="4.5.5" />
+ 		<PackageVersion Include="System.Text.Encodings.Web" Version="8.0.0" />
+ 		<PackageVersion Include="System.Text.Json" Version="8.0.4" />
+ 		<PackageVersion Include="xunit" Version="2.9.0" />
+ 		<PackageVersion Include="xunit.runner.console" Version="2.9.0" />
+ 		<PackageVersion Include="xunit.runner.visualstudio" Version="2.8.2" />
+
```

Comment:
Tiny nit: remove blank line. (I'm happy to do it in a follow-up PR if you want.)

Reply:
Side effect of the script. I'll clean it up.